### DISCUSSION TITLE
How to wait for automatic gates to open

### DISCUSSION THREAD

**Discussion by stev001** [Stephan-S/FS19_AutoDrive#2066] [Q&A] [answered]
I have tried to search for this and unable to find anything.
Is there a way to create a wait point?
Scinario
Im coming to a gate which auto opens and would like to wait for it to open, currently it tries to drive up the gate while it is opening.

Is there a way to add a wait 10 seconds at a specific node (ie. just before the gate) then to carry on

Thanks

**Comment by JustDen1234** [comment, marked as answer]
This is how I do this, slow down the tractor/truck or whatever, so the gate has time to open.
![image](https://user-images.githubusercontent.com/68871402/131865403-b8dfb21d-5117-46cd-bed8-e08242e2fd16.png)

_Note: i'm just a gamer, not a developer of AutoDrive_

**Reply by stev001** [reply]
Doh! Didnt think of that will give it a go.
Thanks
